test/jest/__tests__/boot/ual.spec.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable */
```

Comment:
eslint disable?

Reply:
My bad! I forgot to remove this line.

Fixed